tests/integration/routes/profile/patch-profile-test.js

```diff
@@ -15,7 +15,7 @@ var routeOptions = {
   },
   payload: {
     data: {
-      id: 'userid123-profile',
+      id: 'userid123',
```

Comment:
The `id` was correct for the profile, the `-profile` suffix is there because the resource that gets updated is the profile, not the account, which are separated because a user can pretty much do what they want with profile, while the account is much more restricted. Compare http://docs.accountjsonapi.apiary.io/#reference/current-user/profile/update